src/main/scala/com/microsoft/partnercatalyst/fortis/spark/logging/AppInsightsTelemetry.scala

```diff
@@ -1,29 +1,40 @@
 package com.microsoft.partnercatalyst.fortis.spark.logging
 
+import java.util
+
 import com.microsoft.applicationinsights.{TelemetryClient, TelemetryConfiguration}
-import java.util.HashMap
 
 class AppInsightsTelemetry extends FortisTelemetry {
   private val client: TelemetryClient = new TelemetryClient(TelemetryConfiguration.createDefault())
 
   def logIncomingEventBatch(streamId: String, connectorName: String, batchSize: Long): Unit = {
-    val properties = new HashMap[String, String](2)
+    val properties = new util.HashMap[String, String](2)
```

Comment:
This is to fix a Scala warning. More details: https://stackoverflow.com/q/18196520/3817588